pkg/deploy/spindeploy/save.go

```diff
@@ -36,6 +36,10 @@ func (d *Deployer) deployConfig(ctx context.Context, scheme *runtime.Scheme, gen
 	// the status. But things happen.
 	d.log.Info(fmt.Sprintf("saving %d manifests across %d services", count, len(gen.Config)))
 	for k := range gen.Config {
+		if _, ok := unmanage[k]; ok {
```

Comment:
when saving manifests for each service, it search the service on the unmanage map, if present it skips save object on k8s (applying)

Reply:
It will skip applying the Deployment & Services for the given services. The resources has a concatenation for the service, so they will be applied.